packages/api-explorer/src/JsonForm/index.jsx

```diff
@@ -1,111 +1,109 @@
-import objectCustomEditor from './object-custom-editors'
-import notCustomEditor from './not-custom-editor'
-import anyOfEditor from './anyOf-custom-editor'
-import it from '../../i18n/it.json'
+import configureJsonEditor from './configureJsonEditor'
+import resolveReference from './resolveReference'
 
 import './bootstrap4.css'
 import './custom-bootstrap4.css'
 
-function configureJSONEditor(intl, setFormSubmissionListener) {
-  const editorsKeys = Object.keys(JSONEditor.defaults.editors)
-  const keysToExclude = ['array', 'object', 'not', 'anyOf']
-  editorsKeys.filter(key => !keysToExclude.includes(key)).forEach(key => {
-    JSONEditor.defaults.editors[key] = getCustomEditor(key);
-  });
-
-  JSONEditor.defaults.languages.it = it
-  JSONEditor.defaults.language = intl.locale
-
-  JSONEditor.defaults.editors.array = arrayCustomEditor()
-  JSONEditor.defaults.editors.object = objectCustomEditor()
-  JSONEditor.defaults.editors.not = notCustomEditor()
-  JSONEditor.defaults.editors.anyOf = anyOfEditor(intl,setFormSubmissionListener)
-
-  JSONEditor.defaults.themes.antdTheme = antdTheme
-
-  // eslint-disable-next-line consistent-return
-  JSONEditor.defaults.resolvers.unshift((scheme) => {
-    // If the schema can be of any type
-    if (
-      (scheme.type === "string" &&
-        scheme.media &&
-        scheme.media.binaryEncoding === "base64") ||
-      scheme.format === "binary"
-    ) {
-      return "base64";
-    }
-  });
-
-  // eslint-disable-next-line consistent-return
-  JSONEditor.defaults.resolvers.unshift((scheme) => {
-    if (scheme.not && scheme.not.type) {
-      return 'not'
-    }
-  })
-
-  // eslint-disable-next-line consistent-return
-  JSONEditor.defaults.resolvers.unshift((scheme) => {
-    if (scheme.anyOf) {
-      return 'anyOf'
-    }
-  })
-}
-
 class JsonForm extends Component {
   constructor(props) {
     super(props);
     this.editor = null;
     this.ref = null;
+
+    this.state = {
+      error: null,
+      jsonSchema: null
+    }
+    this.unmounted = false
   }
 
-  createEditor(element) {
-    const {intl, onChange, schema, setFormSubmissionListener} = this.props
-    if (this.editor === null) {
-      const schemaToRender = getSchemaToRender(schema)
+  componentDidMount() {
+    const {schema} = this.props
+    resolveReference(schema)
+      .then(convertedSchema => {
+        if (!this.unmounted) {
+          return this.setState({
+            jsonSchema: convertedSchema
+          })
+        }
+        return null
+      }).catch(err => {
+        if (err && !this.unmounted) {
+          return this.setState({error: err.message})
+        }
+        return null
+      })
+  }
 
-      configureJSONEditor(intl, setFormSubmissionListener)
+  componentWillUnmount() {
+    this.unmounted = true
+  }
+
+  createEditor(element, jsonSchema) {
+    const {intl, onChange, setFormSubmissionListener, title} = this.props
+    if (this.editor === null) {
```

Comment:
Negate the condition and un-indent the rest of the code

Reply:
done